### ISSUE TITLE
[Feature Suggestion] Full charge only attacks

### ISSUE THREAD

**Issue by ChocoMartini** [sf-inc/sihywtcamc#29]
What about a feature that you can attack with weapons in full charge only?

**Comment by charlyb01** [comment]
Yup that's not a bad idea. I may add it in the future =)
I'm not sure it would make combats better but there is only one way to know!

**Comment by ryleu** [comment]
I would LOVE to see this! I have some players who absolutely refuse to wait for the bar to fill and then complain about the game being too hard.

**Comment by charlyb01** [comment]
I might work on an update of this mod soon, I will let you know when the work is done

**Comment by ryleu** [comment]
any progress on this?

**Comment by charlyb01** [comment]
Hello, indeed I forgot to mention it on GitHub!

When starting to work again on this mod, in order to add many other features from combat snapshots, someone mentioned that [Combatify](https://modrinth.com/mod/combatify) is a mod that does exactly this: adding every feature from combat snapshots. As I don't do duplicate mods, I decided to stop adding features from combat snapshots, and I will even remove the ones I already added to the next release that will be for the next Minecraft major version.  
By doing so, this mod will focus on adding only original features to combat. This will be way easier for me to maintain and avoid duplicate functionalities, when one mod is already doing the work correctly.
